contracts/test/TestContracts/StableTokenV3.sol

```diff
@@ -0,0 +1,305 @@
+// // SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
we should update this implementation once we merge your changes to the credit gas function